WTWSMS/localisation/ACR_cybrxkhan_misc.csv

```diff
@@ -223,7 +223,6 @@ ancrel.2511.tooltip.2;\n\nSeems like his love spell worked after all. And there
 ancrel.2511.tooltip.3;\n\nWho knows what other crazy things he might be doing?;\n\nQui sait quelles autres choses folles il pourrait faire ?;;;;;;;;;;x
 ancrel.2511.tooltip.4;\n\nYou gently tell him he is being silly. �Someday you�ll find a good wife,� you tell him in a sisterly tone. �I can help you find one in the future, if you want.�;\n\nVous dites doucement qu'il est stupide. "Un jour, tu trouveras une bonne femme", dites-vous d'un ton fraternel. "Je peux vous aider � trouver une dans un futur proche, si vous voulez.";;;;;;;;;;x
 ancrel.2511.tooltip.5;\n\nYou gently tell him he is being silly. �Someday you�ll find a good wife,� you tell him in a motherly tone. �I can help you find one in the future, if you want.�;\n\nVous dites doucement qu'il est stupide. "Un jour, tu trouveras une bonne femme", dites-vous d'un ton paternel. "Je peux vous aider � trouver une dans un futur proche, si vous voulez.";;;;;;;;;;x
-ancrel.2511.tooltip.5;\n\nYou gently tell her that he is being silly. �Someday you�ll find a good wife,� you tell her in a motherly tone. �I can help you find one in the future, if you want.�;\n\nVous dites doucement qu'elle est stupide. "Un jour, tu trouveras une bonne femme", dites-vous d'un ton paternel. "Je peux vous aider � trouver une dans un futur proche, si vous voulez.";;;;;;;;;;x
```

Comment:
I'm not sure this should be removed outright, rather than being a 6.

Reply:
There's no ancrel.2511.tooltip.6 in the code

Reply:
In ACR it is ancrel.2511.tooltip.6. I would advice that we stick to that.

Reply:
ok, done

Reply:
ok, done